github/orgs_rules_test.go

```diff
@@ -27,7 +27,6 @@ func TestOrganizationsService_GetAllOrganizationRulesets(t *testing.T) {
 			"source_type": "Organization",
 			"source": "o",
 			"enforcement": "active",
-			"bypass_mode": "none",
```

Comment:
You seem only removed incorrect and never added this bypass mode to the new place in test.
Fine for us, may be an issue making such PR upstream.

Reply:
Bypass actors were there before actually. GitHub just moved bypass mode values to a type of bypass actor.